_posts/2024-06-27-WildFly-playing-with-generative-ai.adoc

```diff
@@ -0,0 +1,319 @@
+---
+layout: post
+title:  "Playing with Generative AI with WildFly"
+date:   2024-06-27
+tags: generative-ai rag wildfly
+author: ehugonne
+description: How to try the Generative AI feature pack with WildFly
+---
+
+This blog post provides information on how to use the AI Galleon Feature pack to write generative AI applications.
+Please note that this feature pack is currently a proof of concept and might not be ready for production.
```

Comment:
In this initial section you need to tell people how to get the FP, i.e. its maven coords, if it's been published, or if the idea is they build it themselves it needs a code link and build instructions.

Reply:
I'm waiting for the final release version :)